### PR TITLE
type(update): Updated Litmus-UI to 1.2.0 and added minor UI fixes ⚡️

### PR DESCRIPTION
**Pull request by S-ayanide**
Signed-off-by: Sayan Mondal <sayan@chaosnative.com>

## Proposed changes

* Updated Litmus-UI to 1.2.0
* Added minor UI fixes ⚡️

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [X] New feature (non-breaking change which adds functionality)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [X] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [X] I have signed the commit for DCO to be passed.
- [X] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by SarthakJain26** [comment]
Please do not merge this before this PR gets merged: https://github.com/litmuschaos/litmus/pull/2882

File: litmus-portal/frontend/src/views/Analytics/ApplicationDashboard/Panel/PanelContent.tsx
Comment (whitetiger1399) [inline litmus-portal/frontend/src/views/Analytics/ApplicationDashboard/Panel/PanelContent.tsx:64]: while replacing the `EventMetric` export with `GraphMetric` the `viewEventMetric` hook also got changed. May we please revert it to `veiwEventMetirc`

Comment (ishangupta-ds) [reply litmus-portal/frontend/src/views/Analytics/ApplicationDashboard/Panel/PanelContent.tsx:64]: Yes both were separate @S-ayanide

**Review by ishangupta-ds — CHANGES_REQUESTED** [review]

File: litmus-portal/frontend/src/views/CreateWorkflow/TuneWorkflow/WorkflowPreview/styles.ts
Comment (whitetiger1399) [inline litmus-portal/frontend/src/views/CreateWorkflow/TuneWorkflow/WorkflowPreview/styles.ts:22]: As this is a workflow file so it should have colors from `status/workflow` and not `status/experiment`. May you please check with the design for the color and if required we may also do changes in the `status/workflow` for the required hex

File: litmus-portal/frontend/src/views/Analytics/ApplicationDashboard/Panel/PanelContent.tsx
Comment (S-ayanide) [reply litmus-portal/frontend/src/views/Analytics/ApplicationDashboard/Panel/PanelContent.tsx:64]: Yes sure, reverting this change

File: litmus-portal/frontend/src/views/Analytics/ApplicationDashboard/Panel/PanelContent.tsx
Comment (S-ayanide) [reply litmus-portal/frontend/src/views/Analytics/ApplicationDashboard/Panel/PanelContent.tsx:279]: The Graph Metric state won't be there in this file right!

Comment (whitetiger1399) [reply litmus-portal/frontend/src/views/Analytics/ApplicationDashboard/Panel/PanelContent.tsx:279]: Yes. seViewEventMetric should be the one.

**Review by amityt — APPROVED** [review]

**Review by whitetiger1399 — APPROVED** [review]

**Review by ishangupta-ds — APPROVED** [review]

**Pull request merged**
